usaspending_api/disaster/v2/views/award/count.py

```diff
@@ -0,0 +1,36 @@
+from django.db.models import OuterRef, Q, Exists
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from usaspending_api.common.cache_decorator import cache_response
+from usaspending_api.disaster.v2.views.count_base import CountBase
+from usaspending_api.awards.models import FinancialAccountsByAwards
+from usaspending_api.awards.models import Award
+
+
+class AwardCountViewSet(CountBase):
+    """
+    Obtain the count of Agencies related to supplied DEFC filter.
+    """
+
+    endpoint_doc = "usaspending_api/api_contracts/contracts/v2/disaster/award/count.md"
+
+    required_filters = ["def_codes", "award_type_codes"]
+
+    @cache_response()
+    def post(self, request: Request) -> Response:
+        filters = [
+            Q(award=OuterRef("pk")),
+            self.is_in_provided_def_codes(),
+            self.all_closed_defc_submissions,
+            self.is_non_zero_award_cpe(),
+        ]
+        count = (
+            Award.objects.annotate(include=Exists(FinancialAccountsByAwards.objects.filter(*filters).values("pk")))
+            .filter(include=True)
+            .filter(self.is_provided_award_type())
```

Comment:
the two filter lines can be combined into a single .filter()

Reply:
Unless there's a python trick I've missed, I can't combine the keyword argument with the positional one.

Reply:
Does `.filter(self.is_provided_award_type(), include=True)` not work?